src/Node/DelimitedList.php

```diff
@@ -31,4 +31,17 @@ public function addToken($node) {
         }
         $this->children[] = $node;
     }
+
+    /**
+     * @return Node[]
+     */
+    public function getListElements() {
+        $listElements = [];
+        foreach ($this->children as $child) {
+            if($child instanceof Node) {
+                $listElements[] = $child;
```

Comment:
In #103 we should parse `QualifiedNameList` with `TokenKind::BarToken` delimiter, but in most cases its `TokenKind::CommaToken`.
But its probable I just misunderstood your suggestion ("simply define the delimiter as a constant on DelimitedList") :smile: